src/screens/ConstructionSiteOverviewScreen.js

```diff
@@ -36,9 +35,9 @@ export const ConstructionSiteOverviewScreen = ({ navigation }) => {
           : `${formattedStartDate} - ${formattedEndDate}`;
 
       const propItem = {
-        title,
+        title: title,
```

Comment:
just `title` would be straight forward, right?

Reply:
✔️